torch/csrc/autograd/functions/convolution.cpp

```diff
@@ -366,7 +413,25 @@ auto ConvBackward::apply(const variable_list& grad_outputs) -> variable_list {
     should_compute_output(2) && bias.defined(),
   };
 
-  if (use_cudnn) {
+  if (use_depthwise) {
+    if (output_mask[0] || output_mask[1]) {
+      auto kernel_size = weight.sizes().slice(2);
+      auto stride = vecToInt64(this->stride);
+      auto padding = vecToInt64(this->padding);
+      auto dilation = vecToInt64(this->dilation);
+
+      std::tie(grad_input, grad_weight) = at::conv_depthwise2d_backward(
+          grad_output, input, weight, kernel_size, stride, padding, dilation,
+          {output_mask[0], output_mask[1]});
+      }
+
+      // THCUNN implementation does not handle bias, so we do it ourselves
+      if (output_mask[2]) {
+        grad_bias = bias.type().tensor();
+        grad_bias.resize_as_(bias).zero_();
+        update_grad_bias(grad_output, grad_bias);
```

Comment:
Instead of creating a zeroed tensor and accumulating into it, just compute the grad_bias from `grad_output` i.e.

```
grad_bias = compute_grad_bias(grad_output);
```